src/main/scala/uk/gov/nationalarchives/DiscoveryService.scala

```diff
@@ -20,10 +20,16 @@ import scala.xml.XML
 class DiscoveryService(discoveryBaseUrl: String, backend: SttpBackend[IO, Fs2Streams[IO]], randomUuidGenerator: () => UUID) {
   private val folder = Folder
 
+  private def replaceUnicodeDecimals(input: String) =
```

Comment:
Maybe `replaceHtmlCodesWithUnicodeChars`, `convertHtmlCodesWithUnicodeChars`, `replaceDecimalCharsWithUnicodeChars`, `replaceDecimalCharsWithUnicode`, `convertDecimalCharsToUnicodeChars` or `convertDecimalCharsToUnicode`?

Reply:
I've used `replaceHtmlCodesWithUnicodeChars` as it's a short function but I think we need to be careful with these long names as we end up with a lot of line breaks which can be quite hard to read.